ENIGMAsystem/SHELL/Graphics_Systems/OpenGL1/GLtextures_impl.h

```diff
@@ -0,0 +1,40 @@
+/** Copyright (C) 2008-2013 Josh Ventura
+*** Copyright (C) 2013,2019 Robert B. Colton
+***
+*** This file is a part of the ENIGMA Development Environment.
+***
+*** ENIGMA is free software: you can redistribute it and/or modify it under the
+*** terms of the GNU General Public License as published by the Free Software
+*** Foundation, version 3 of the license or any later version.
+***
+*** This application and its source code is distributed AS-IS, WITHOUT ANY
+*** WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+*** FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+*** details.
+***
+*** You should have received a copy of the GNU General Public License along
+*** with this code. If not, see <http://www.gnu.org/licenses/>
+**/
+
+#ifdef INCLUDED_FROM_SHELLMAIN
+#  error This file includes non-ENIGMA STL headers and should not be included from SHELLmain.
+#endif
+
+#ifndef ENIGMA_GL_TEXTURES_IMPL_H
+#define ENIGMA_GL_TEXTURES_IMPL_H
+
+#include "Graphics_Systems/General/GStextures_impl.h"
+#include "Graphics_Systems/General/OpenGLHeaders.h"
+
+namespace enigma {
+
+struct GL1Texture : Texture {

```

Comment:
Do we not have a directory for common files between GL1/3?

Reply:
No, shall I create one? I can do that and was already thinking of it.

Reply:
Ok: d72a667af40173d090571b96bd57ab4df248a299

That's a good idea, let's just get it out of the way now. Later when I bring back `D3DCompile` based user shaders, I will be creating a `Direct3D` folder since that code is going to be common to the D3D systems.